tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -284,7 +284,7 @@ public async Task AddMicrosoftWebAppCallsWebApi_WithConfigActionParameters()
 
             var builder = services.AddAuthentication()
                 .AddMicrosoftWebApp(_configureMsOptions, null, OidcScheme)
-                .CallsWebApi(_configureAppOptions, initialScopes);
+                .CallsWebApis(_configureAppOptions, initialScopes);
```

Comment:
See, in this case, if we had chosen the other design, we'd have to pass-in the _configureAppOptions in all the CallsMicrosoftGraph and CallsDownstreamApis, with a risk of errors (pass-in different things, whereas this would be the same service, really), and duplication of code.